src/content/docs/mobile-apps/ios-app/introduction-ios-mobile-app.mdx

```diff
@@ -18,9 +18,7 @@ The user interface for New Relic's iPhone and iPad app provides functionality si
 
 ## Features [#dashboard]
 
->New Relic's iOS app includes these New Relic products and features:
-
-New Relic's iOS app for iPhone and iPad includes these New Relic products and features:
+New Relic's iOS app includes these New Relic products and features:
```

Comment:
I thought we were supposed to stop referring to things like APM, browser monitoring, etc. as products and instead refer to them something along the lines of "capabilities."